autogluon/task/text_classification/dataset.py

```diff
@@ -119,28 +121,79 @@ def get_dataset(self, segment='train'):
         sampler, _ = get_split_samplers(dataset, split_ratio=0.2)
         return SampledDataset(dataset, sampler)
 
-class TSVClassificationTask(AbstractGlueTask):
-    def __init__(self, *args, **kwargs): # passthrough arguments to TSVDataset
-        # (filename, field_separator=nlp.data.Splitter(','), num_discard_samples=1, field_indices=[2,1])
-        self.args = args
-        self.kwargs = kwargs
-        is_pair = False
-        class_labels = ['0', '1']
-        metric = CompositeEvalMetric()
-        metric.add(F1())
-        metric.add(Accuracy())
-        super(TSVClassificationTask, self).__init__(class_labels, metric, is_pair)
-        dataset = nlp.data.TSVDataset(*self.args, **self.kwargs)
-        # do the split
-        train_sampler, val_sampler = get_split_samplers(dataset, split_ratio=0.8)
-        self.trainset = SampledDataset(dataset, train_sampler)
-        self.valset = SampledDataset(dataset, val_sampler)
+class CustomTSVClassificationTask(AbstractGlueTask):
+    """
+
+    Parameters
+    ----------
+    filepath: str, path of the file
+        Any valid string path is acceptable.
+    sep : str
+        Delimiter to use.
+    header : int, list of int, default 'infer'
+        Row number(s) to use as the column names, and the start of the
+        data.
+    names : array-like, optional
+        List of column names to use.
+    index_col : int, str, sequence of int / str, or False, default ``None``
+    usecols : list-like or callable, optional
+        Return a subset of the columns.
+    For our keyword argument, plesase see
```

Comment:
Typo and please rephrase to:

'For additional keyword arguments related to loading the data, please see'